docs/sidebars.js

```diff
@@ -53,7 +53,8 @@ module.exports = {
 	    "cli/choose-a-cluster",
 	    "cli/transfer-tokens",
 	    "cli/delegate-stake",
-	    "cli/manage-stake-accounts",
+            "cli/manage-stake-accounts",
+            "cli/deploy-a-program",
```

Comment:
Some extra indents here

Reply:
Dan tabs!

Reply:
Lol I feel ya, but @danpaul000's last name is not "Tabs".  Just FYI